lib/backend/k8s/resources/affinityblock.go

```diff
@@ -1,174 +0,0 @@
-// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
```

Comment:
We can't just delete this outright if we still want to support `host-local` IPAM as well.

We will need some way to trigger the code to use this representation of affinity for host-local, and the datastore-driven one when using calico-ipam.